.eslintrc

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
why remove this?

Reply:
in fact, if you being it back and ask your editor to format the files, everything will look much nicer